draft-ietf-quic-transport.md

```diff
@@ -294,6 +296,15 @@ QUIC version negotiation allows for multiple versions of the protocol to be
 deployed and used concurrently. Version negotiation is described in
 {{version-negotiation}}.
 
+## Support for Latency Monitoring
+
+Network administrators often find difficult to monitor the behavior
+of encrypted protocols. Bandwidth can be monitored by observing the
+flow of encrypted packets, but the usal tools for monitoring
+latency require observing packet numbers and acknowledgements, which
+in QUIC are encrypted.
+QUIC packets include a clear text latency spin bit that enables
```

Comment:
nit: "QUIC packets enable monitoring of latency by including a clear text latency spin bit that is reflected between peers."